OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -325,6 +326,7 @@
     <Compile Include="Traits\Crushable.cs" />
     <Compile Include="Traits\CustomSellValue.cs" />
     <Compile Include="Traits\CustomSelectionSize.cs" />
+    <Compile Include="Traits\CustomRenderSize.cs" />
```

Comment:
Is there a reason this isn't in `Traits\Render\` with `AutoRenderSize`?

Reply:
Mostly that I wanted to have the Render counterpart in the same folder as the Selection trait, and `CustomSelectionSize` is now really unrelated to rendering.